CONTRIBUTING.md

```diff
@@ -42,6 +42,11 @@ Before getting started, you should install some developer tools. These are not
 required to deploy Conjur but they will let you develop using a standardized,
 expertly configured environment.
 
+
+##### Preventing Secret Leaks
```

Comment:
looks out of place to me. how can it be before you install git?

also - why are there 4 `#`?